src/styles/page-style.scss

```diff
@@ -0,0 +1,30 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
Only `box-sizing` property allows for `*` selector
```suggestion
  box-sizing: border-box;
```

Reply:
@loralevitska, thanks a lot.